CallAutomation_SimpleIvr/Program.cs

```diff
@@ -126,66 +111,41 @@ await client.GetCallConnection(@event.CallConnectionId)
             }
             else
             {
-                PlaySource invalidAudio = new FileSource(new Uri(audioBaseUrl + builder.Configuration["InvalidAudio"]));
+                PlaySource invalidAudio = new FileSource(new Uri(baseUri + builder.Configuration["InvalidAudio"]));
                 await client.GetCallConnection(@event.CallConnectionId).GetCallMedia().PlayToAllAsync(invalidAudio, audioPlayOptions);
```

Comment:
`await client.GetCallConnection(@event.CallConnectionId).GetCallMedia()` there are too many of these you dont need to do this everytime. 
Add a variable at top something like `var callMedia = client.GetCallConnection(@event.CallConnectionId).GetCallMedia();` and then use it to call play and recognize api.

Reply:
Something like this
<img width="544" alt="image" src="https://user-images.githubusercontent.com/20677492/221986068-bda7f1be-35ed-404d-a0d7-3b588c1c03f8.png">